<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FITMainWindow</class>
 <widget class="QMainWindow" name="FITMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GEMSFITS v.0.1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splV">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="widgetList" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splKey">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="widgetKeys" native="true">
           <layout class="QGridLayout" name="gridLayout" rowstretch="0">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QToolButton" name="filterButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../fgui.qrc">
                <normaloff>:/menu/Icons/SelectDisplayRecordIcon.png</normaloff>:/menu/Icons/SelectDisplayRecordIcon.png</iconset>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="filterEdit"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widgetQuery" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Search</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="queryEdit"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widgetRecord" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splRecord">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Edit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="recordEdit"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Status</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="statusEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widgetGEMS" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splGEMS">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="widgetIC" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>  IComp (Elements)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableIComp">
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustToContents</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="dragEnabled">
               <bool>true</bool>
              </property>
              <property name="defaultDropAction">
               <enum>Qt::CopyAction</enum>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="rowCount">
               <number>2</number>
              </property>
              <property name="columnCount">
               <number>5</number>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>40</number>
              </attribute>
              <attribute name="horizontalHeaderMinimumSectionSize">
               <number>10</number>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderDefaultSectionSize">
               <number>20</number>
              </attribute>
              <row/>
              <row/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widgetPhase" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Phases/Components</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="listPhases">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="tabKeyNavigation">
               <bool>true</bool>
              </property>
              <property name="dragEnabled">
               <bool>true</bool>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTask">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Select_project"/>
    <addaction name="action_New_Project"/>
    <addaction name="action_Config_Project"/>
    <addaction name="separator"/>
    <addaction name="action_DataBase_mode"/>
    <addaction name="action_Task_Mode"/>
    <addaction name="actionFits_View_Mode"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_GEMS"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuRecord">
    <property name="title">
     <string>&amp;Record</string>
    </property>
    <addaction name="actionCreate_New"/>
    <addaction name="actionUpdate_Test"/>
    <addaction name="separator"/>
    <addaction name="action_Update"/>
    <addaction name="action_Insert"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="action_Find"/>
   </widget>
   <widget class="QMenu" name="menuRecod_List">
    <property name="title">
     <string>&amp;Database</string>
    </property>
    <addaction name="actionOverwrite"/>
    <addaction name="separator"/>
    <addaction name="actionBackup_to_JSON"/>
    <addaction name="actionRestore_from_JSON"/>
    <addaction name="separator"/>
    <addaction name="actionRestore_from_csv"/>
    <addaction name="actionBackup_to_csv"/>
    <addaction name="separator"/>
    <addaction name="actionRestore_from_YAML"/>
    <addaction name="actionBackup_to_YAML"/>
    <addaction name="separator"/>
    <addaction name="actionRestore_from_TXT"/>
    <addaction name="actionBackup_to_TXT"/>
    <addaction name="separator"/>
    <addaction name="action_Delete_multiple_data"/>
   </widget>
   <widget class="QMenu" name="menu_Calc">
    <property name="title">
     <string>&amp;Calculate</string>
    </property>
    <addaction name="action_Run_test"/>
    <addaction name="action_Show_Results"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Preferences"/>
    <addaction name="separator"/>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionAuthors"/>
    <addaction name="actionLicense"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionData_in_JSON"/>
    <addaction name="actionData_in_YAML"/>
   </widget>
   <addaction name="menuTask"/>
   <addaction name="menuRecord"/>
   <addaction name="menuRecod_List"/>
   <addaction name="menu_Calc"/>
   <addaction name="menuView"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBarTask">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBarMenu">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
   <addaction name="action_Find"/>
   <addaction name="separator"/>
   <addaction name="actionCreate_New"/>
   <addaction name="actionUpdate_Test"/>
   <addaction name="action_Insert"/>
   <addaction name="action_Update"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="action_Run_test"/>
   <addaction name="action_Show_Results"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBarGems">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_DataBase_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../fgui.qrc">
     <normaloff>:/menu/Icons/DatabaseModeIcon.png</normaloff>:/menu/Icons/DatabaseModeIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Database Mode</string>
   </property>
  </action>
  <action name="action_Task_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../fgui.qrc">
     <normaloff>:/modules/Icons/DualThModuleIcon.png</normaloff>:/modules/Icons/DualThModuleIcon.png</iconset>
   </property>
   <property name="text">
    <string>Fit &amp;Task Mode</string>
   </property>
  </action>
  <action name="action_New_Project">
   <property name="text">
    <string>&amp;New Project...</string>
   </property>
  </action>
  <action name="action_Select_project">
   <property name="text">
    <string>&amp;Open Project...</string>
   </property>
  </action>
  <action name="actionSelect_GEMS">
   <property name="text">
    <string>Select &amp;GEMS3K Files</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionCreate_New">
   <property name="icon">
    <iconset resource="../fgui.qrc">
     <normaloff>:/menu/Icons/NewCreateRecordIcon.png</normaloff>:/menu/Icons/NewCreateRecordIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Create (New)...</string>
   </property>
   <property name="toolTip">
    <string>Create New by template</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="icon">
    <iconset resource="../fgui.qrc">
     <normaloff>:/menu/Icons/SelectDisplayRecordIcon.png</normaloff>:/menu/Icons/SelectDisplayRecordIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Find...</string>
   </property>
   <property name="toolTip">
    <string>Find</string>
   </property>
  </action>
  <action name="action_Update">
   <property name="icon">
    <iconset resource="../fgui.qrc">
     <normaloff>:/menu/Icons/SaveCurrentRecordIcon.png</normaloff>:/menu/Icons/SaveCurrentRecordIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Update ...</string>
   </property>
   <property name="toolTip">
    <string>Update current record</string>
   </property>
  </action>
  <action name="action_Insert">
   <property name="icon">
    <iconset resource="../fgui.qrc">
     <normaloff>:/menu/Icons/NewCloneRecordIcon.png</normaloff>:/menu/Icons/NewCloneRecordIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Insert ...</string>
   </property>
   <property name="toolTip">
    <string>Insert new record</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../fgui.qrc">
     <normaloff>:/menu/Icons/DeleteCurrentRecordIcon.png</normaloff>:/menu/Icons/DeleteCurrentRecordIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete ...</string>
   </property>
   <property name="toolTip">
    <string>Delete current record</string>
   </property>
  </action>
  <action name="actionUpdate_Test">
   <property name="icon">
    <iconset resource="../fgui.qrc">
     <normaloff>:/menu/Icons/CalculateBCCIcon.png</normaloff>:/menu/Icons/CalculateBCCIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Test Edited Record Format</string>
   </property>
  </action>
  <action name="actionBackup_to_JSON">
   <property name="text">
    <string>Backup data to &amp;JSON...</string>
   </property>
  </action>
  <action name="actionRestore_from_JSON">
   <property name="text">
    <string>Restore data from JSO&amp;N...</string>
   </property>
  </action>
  <action name="actionBackup_to_csv">
   <property name="text">
    <string>Export data to &amp;CSV...</string>
   </property>
  </action>
  <action name="actionRestore_from_csv">
   <property name="text">
    <string>Import data from C&amp;SV...</string>
   </property>
  </action>
  <action name="actionBackup_to_TXT">
   <property name="text">
    <string>Export fit task &amp;TXT file...</string>
   </property>
  </action>
  <action name="actionRestore_from_TXT">
   <property name="text">
    <string>Import fit task T&amp;XT file...</string>
   </property>
  </action>
  <action name="actionBackup_to_YAML">
   <property name="text">
    <string>Export data to &amp;YAML...</string>
   </property>
  </action>
  <action name="actionRestore_from_YAML">
   <property name="text">
    <string>Import data from Y&amp;AML...</string>
   </property>
  </action>
  <action name="action_Run_test">
   <property name="icon">
    <iconset resource="../fgui.qrc">
     <normaloff>:/menu/Icons/CalculateRecordDataIcon.png</normaloff>:/menu/Icons/CalculateRecordDataIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Run Fitting Task</string>
   </property>
  </action>
  <action name="action_Show_Results">
   <property name="icon">
    <iconset resource="../fgui.qrc">
     <normaloff>:/menu/Icons/ShowGEMResultsIcon.png</normaloff>:/menu/Icons/ShowGEMResultsIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Show Task Results... </string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="text">
    <string>&amp;Preferences ...</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../fgui.qrc">
     <normaloff>:/menu/Icons/ShowHelpWindowIcon.png</normaloff>:/menu/Icons/ShowHelpWindowIcon.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAuthors">
   <property name="text">
    <string>Authors</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="../fgui.qrc">
     <normaloff>:/menu/Icons/DisplayNextRecordIcon.png</normaloff>:/menu/Icons/DisplayNextRecordIcon.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="../fgui.qrc">
     <normaloff>:/menu/Icons/DisplayPrevRecordIcon.png</normaloff>:/menu/Icons/DisplayPrevRecordIcon.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </action>
  <action name="action_Config_Project">
   <property name="text">
    <string>&amp;Reconfigure Project...</string>
   </property>
  </action>
  <action name="actionData_in_JSON">
   <property name="text">
    <string>Data in &amp;JSON</string>
   </property>
  </action>
  <action name="actionData_in_YAML">
   <property name="text">
    <string>Data in &amp;YAML</string>
   </property>
  </action>
  <action name="actionOverwrite">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Overwrite</string>
   </property>
  </action>
  <action name="action_Delete_multiple_data">
   <property name="text">
    <string>&amp;Delete multiple data... </string>
   </property>
  </action>
  <action name="actionFits_View_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../fgui.qrc">
     <normaloff>:/modules/Icons/FitSpaceModuleIcon.png</normaloff>:/modules/Icons/FitSpaceModuleIcon.png</iconset>
   </property>
   <property name="text">
    <string>Fit &amp;View Mode</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../fgui.qrc"/>
 </resources>
 <connections/>
</ui>
